Riot/Modules/Room/RoomViewController.m

```diff
@@ -4998,10 +4998,11 @@ - (void)roomInputToolbarView:(MXKRoomInputToolbarView*)toolbarView heightDidChan
     if (self.roomInputToolbarContainerHeightConstraint.constant != height)
     {
         // Hide temporarily the placeholder to prevent its distortion during height animation
-        if (!savedInputToolbarPlaceholder)
+        if (toolbarView.placeholder)
         {
-            savedInputToolbarPlaceholder = toolbarView.placeholder.length ? toolbarView.placeholder : @"";
+            savedInputToolbarPlaceholder = toolbarView.placeholder;
         }
+        
         toolbarView.placeholder = nil;
```

Comment:
This code was a bit unreliable, sometimes it removed the placeholder text when the height of it did change, without restoring it properly.
In reality I think most of the code of this function can safely be removed.

I actually tested removing this code alongside lines 5014 to 5020, and actually the placeholder works fine since both in the new composer and the old composer it uses the `redraw` `contentMode`.
And it looked even better on both composers, since the placeholder did even stop blinking

Can I actually just remove it?
@langleyd